mobile/lib/repositories/user.repository.dart

```diff
@@ -1,17 +1,23 @@
 import 'package:http/http.dart' as http;
 import 'package:triggo/api/call.dart';
 import 'package:triggo/api/response.dart';
+import 'package:triggo/repositories/credentials.repository.dart';
 import 'package:triggo/repositories/dtos/user.dtos.dart';
 
 class UserRepository {
   final http.Client? client;
+  final CredentialsRepository? credentialsRepository;
```

Comment:
Why it's a possibly null variable ?
```suggestion
  final CredentialsRepository credentialsRepository;
```
Because even in your constructor you `required` this arg.

Reply:
You're right